.ci/install_footloose.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o nounset
+set -o pipefail
+
+cidir=$(dirname "$0")
+arch=$("${cidir}"/kata-arch.sh -d)
+source "${cidir}/lib.sh"
+
+if [ "$arch" != "x86_64" ]; then
+	warn "Footloose only available with x86_64."
+	exit 1
```

Comment:
I'm not sure, but I think @grahamwhaley was implying we should `exit 0` here?

Reply:
@jodh-intel changes applied